console/src/main/java/org/matrix/console/Matrix.java

```diff
@@ -90,22 +91,22 @@ public synchronized MXSession getDefaultSession() {
             return sessions.get(0);
         }
 
-        ArrayList<Credentials> credsList = mLoginStorage.getCredentialsList();
+        ArrayList<HomeserverConnectionConfig> cconfigList = mLoginStorage.getCredentialsList();
```

Comment:
Glancing at `cconfigList` makes me think "TYPO!" - can we call this something which doesn't trip my spelling OCD? `connConfigList` perhaps?